bot/exts/evergreen/gurkan.py

```diff
@@ -0,0 +1,29 @@
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+
+class Gurkan(commands.Cog):
+    """Commands related to Gurkan and the Gurkult."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="gurkancount", aliases=("gc", "gurkcount"))
+    async def gurkancount(self, ctx: commands.Context) -> None:
+        """Get the amount of members with 'gurkan' in their names."""
+        gurkans = [1 for member in ctx.guild.members if "gurkan" in member.display_name.lower()]
+        gurkan_count = sum(gurkans)
```

Comment:
Maybe something like this instead? I'm not sure how I feel about having two variables for this.
```suggestion
        gurkan_count = sum(
            1 for member in ctx.guild.members if "gurkan" in member.display_name.lower()
        )
```

Reply:
Looks much better. I didn't quite like having all of that on a single line nor creating two variables for it, so this solution looks better in some sense.

The only real reason that having two variables would be worth considering is if we would like to add some feature to do additional filtering in the future. That can be added when it becomes relevant, though, so this change looks good to me.